Automattic/wp-calypso
# Assembler - Add the Pick your style step

Closes #80687 #80725

## Proposed Changes

- In `Design your own` step:
  * The Header drawer is opened by default.
  * Users cannot click on the `Pick your style` button unless they have selected any pattern.
    * Show tooltip "Add your first pattern to get started." when the button is disabled
  - Revert the category list scroll introduced in https://github.com/Automattic/wp-calypso/pull/80879

- In `Pick your style` step:
  * The color drawer is opened by default.
  * Users can skip this step by clicking on the `Save and continue` button.


|BEFORE|AFTER|
|--|--|
|<video src="https://github.com/Automattic/wp-calypso/assets/1881481/3bb47efc-81e2-4caf-8c7c-30cfc961df51">|<video src="https://github.com/Automattic/wp-calypso/assets/1881481/4f45a95b-9fe4-4652-8dfb-2c8eef7943f4">|






## Testing Instructions



* Access the assembler `/setup/with-theme-assembler/patternAssembler?siteSlug={ SITE }`
- Click `Pick your style` to verify it's disabled and you see the tooltip
- Click on `Header`, `Sections`, and `Footer` to add patterns 
- Click on `Pick your style`
- Click the back button `<`
- Click on `Pick your style` again and choose a color and font variation
- Click `Save and continue`

Also, verify new interactive elements have Tracks events.

## Pre-merge Checklist
